catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/impl/CatalogFrameworkImplTest.java

```diff
@@ -325,12 +326,26 @@ public void setup()
     federatedSources = createDefaultFederatedSourceList(true);
 
     MimeTypeResolver mimeTypeResolver = mock(MimeTypeResolver.class);
-    MimeTypeToTransformerMapper mimeTypeToTransformerMapper =
-        mock(MimeTypeToTransformerMapper.class);
-    InputTransformer inputTransformer = mock(InputTransformer.class);
-    when(inputTransformer.transform(any(InputStream.class))).thenReturn(new MetacardImpl());
-    when(mimeTypeToTransformerMapper.findMatches(any(Class.class), any(MimeType.class)))
-        .thenReturn(Collections.singletonList(inputTransformer));
+
+    transform = mock(Transform.class);
+
+    when(transform.transform(
+            any(MimeType.class),
+            any(Supplier.class),
+            any(String.class),
+            any(File.class),
+            any(String.class),
+            any(Map.class)))
+        .thenAnswer(
+            new Answer<List<Metacard>>() {
+              @Override
+              public List<Metacard> answer(InvocationOnMock invocationOnMock) throws Throwable {
```

Comment:
✏️  i think this can be simplified to a lambda

Reply:
done